### PR TITLE
Add README

### LINKED ISSUE #13: update README.md

### PR DESCRIPTION
**Pull request by galta95**
closes #13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,37 @@
...
+import Apiv6 from '@map-colonies/node-osm-api';
+
+const api = new Apiv6('https://www.openstreetmap.org', USER_NAME, PASSWORD);
+```
+
+## Installation
+
+```
+$ npm install @map-colonies/node-osm-api
+```
```
Comment (syncush) [inline README.md:15]: indicate it is a shell script, https://docs.github.com/en/free-pro-team@latest/github/writing-on-github/creating-and-highlighting-code-blocks

File: README.md [hunk 2]
```diff
@@ -1,3 +1,37 @@
...
+
+## Usage
+
+- Right now we only support create/upload/close changesets
+
+Create changeset:
+
+```
+api.createChangeset(XML_AS_STRING).then(console.log).catch(console.error);
+```
```
Comment (syncush) [inline README.md:25]: typescript

File: README.md [hunk 3]
```diff
@@ -1,3 +1,37 @@
...
+
+Upload changeset:
+
+```
+api.uploadChangeset(CHANGESET_ID, OSM_CHANGE_FILE).then(console.log).catch(console.error);
+```
```
Comment (syncush) [inline README.md:31]: same

File: README.md [hunk 4]
```diff
@@ -1,3 +1,37 @@
...
+
+Close changeset:
+
+```
+api.closeChangeset(CHANGESET_ID).then(console.log).catch(console.error);
+```
```
Comment (syncush) [inline README.md:37]: same

File: README.md (hunk 2 above)
Comment (syncush) [inline README.md:25]: `.then(console.log).catch(console.error)` is not very informative, 

what is the result of the function?
which errors can be thrown if something goes wrong?

Comment (syncush) [inline README.md:25]: We need a full code snippet, not just function invocation, we want the import, the init of the apiv6 instance, and the invocation. look at [lodash](https://github.com/lodash/lodash) for example

File: README.md [hunk 5]
```diff
@@ -1,3 +1,37 @@
 # node-osm-api
 
-easy communication with osm api
+Easy nodejs communication with OpenStreetMap api
+
+```
+import Apiv6 from '@map-colonies/node-osm-api';
+
+const api = new Apiv6('https://www.openstreetmap.org', USER_NAME, PASSWORD);
```
Comment (syncush) [inline README.md:8]: indicate that it is a typescript

**Review by syncush — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (galta95) [reply README.md]: done

File: README.md (hunk 5 above)
Comment (galta95) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (galta95) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (galta95) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (galta95) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (galta95) [reply README.md]: done

Comment (galta95) [reply README.md]: done

**Review by syncush — APPROVED** [review]

**Pull request merged**